src/test/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesSchedulerIT.java

```diff
@@ -195,12 +195,12 @@ public void testSchedulerPropertiesMerge() {
 
 		assertThat(mergedProperties
 				.get(KubernetesDeployerProperties.KUBERNETES_DEPLOYER_PROPERTIES_PREFIX + ".imagePullPolicy"))
-						.as("Expected value from Scheduler properties, but found in Deployer properties")
-						.isEqualTo("Never");
+				.as("Expected value from Scheduler properties, but found in Deployer properties")
```

Comment:
Typically we try to not do widespread formatting changes in a PR. 
If it must be done, then it is best to do it in a separate commit to help decipher the actual change from the noise. 
In this case, thank you for updating the formatting here as it really needed it and your changes are small enough that it is not an issue. Just a request for the future - thats all.

Reply:
I'm sorry about that. I'll remember and and practice it.

Reply:
Oh no worries - nothing to apologize for. Someone passed that on to me at some point and I am passing it onto you. You can then pass it on to someone as well. 👍🏻